lib/persister/game_data.rb

```diff
@@ -19,82 +35,77 @@ def types
         ]
     end
 
-    private
-    attr_accessor :game
-
     def board_presenter
         {
             name: :board_presenter,
-            data: game.board_presenter.presenter.class.name
+            data: game && game.board_presenter.presenter.class.name
         }
     end
 
     def status
         {
             name: :status,
-            data: game.status,
+            data: game && game.status,
```

Comment:
game&.status http://mitrev.net/ruby/2015/11/13/the-operator-in-ruby/

Reply:
would this: `data: game&.game_state&.board&.positions` still be better than `data: game && game.game_state.board&.positions` considering they would all need to change?

Reply:
safe navigator is easier to read and cleaner than doing &&. If I saw that in a code-base, I'd correct it all to use the &

Reply:
I prefer chaining the safe navigator operator than doing that hard && check when possible.

Reply:
gotcha. Will update